From 039d5faacc5d6877cf7292806cf0d457dc98df2c Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 22 May 2018 21:29:51 +0200
Subject: [PATCH 1/1] x86: adjust addresses in u-boot-spl.lds

The load address for the U-Boot image is 0xfffd0000.

We should not load the .got section to 0x1200000 because this results in
a 4GB SPL image.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 arch/x86/cpu/u-boot-spl.lds | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/cpu/u-boot-spl.lds b/arch/x86/cpu/u-boot-spl.lds
index 8a38d58f123..75be5cf5db8 100644
--- a/arch/x86/cpu/u-boot-spl.lds
+++ b/arch/x86/cpu/u-boot-spl.lds
@@ -39,7 +39,7 @@ SECTIONS
 
         _image_binary_end = .;
 
-	. = 0x120000;
+	. = 0xffff0000;
 	.bss (OVERLAY) : {
 		__bss_start = .;
 		*(.bss*)
-- 
2.17.0

